packages/docs/src/page-configs/ui-elements/select/examples/Slots.vue

```diff
@@ -5,6 +5,7 @@
       label="Prepend slot"
       :options="options"
       v-model="value"
+      bordered
```

Comment:
```suggestion
```

There are no reasons to change style option in slots example.

Reply:
removed this option